tests/executor/ray_run.sh

```diff
@@ -0,0 +1,68 @@
+set -x
```

Comment:
It is not appropriate to place a complete model inference script directly under the `tests` directory.

I suggest placing the Ray example scripts under the `examples/ray` directory.